packages/eslint-config-sheriff/src/getAstroConfig.ts

```diff
@@ -60,6 +60,15 @@ export const getAstroConfig = (
       },
       rules: getTsNamingConventionRule({ isTsx: true }),
     },
+    {
+      files: ['**/*.astro/*.ts'], // Virtual files created by eslint-plugin-astro.
+      languageOptions: {
+        parserOptions: {
+          parser: tseslint.parser,
+          project: customTSConfigPath || true,
+        },
+      },
+    },
```

Comment:
This is required as `eslint-plugin-astro` creates virtual files for eslint for the `script` body.